README.md

```diff
@@ -17,17 +17,12 @@
 <br>
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
 [![PRs Welcome][pr-badge]][pr-url]
-<br>
+[![All Contributors][all-contributors-badge]](#contributors-)
+<br />
 [![Watch on Github][gh-watchers-badge]][gh-watchers-url]
 [![Star on Github][gh-stars-badge]][gh-stars-url]
 [![Tweet][tweet-badge]][tweet-url]
 
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```

Comment:
As this comment claims: we should not modify this section manually as it's updated by all-contributors bot.

Reply:
It's only updated if we have this comment

Since we're not updating the number manually anymore, but are using the link that automatically increases the number for us, we can delete this
https://img.shields.io/github/all-contributors/testing-library/eslint-plugin-testing-library?color=orange&style=flat-square

Reply:
Oh, TIL